### ISSUE TITLE
There is a bug about attention mask

### ISSUE THREAD

**Issue by Jason941** [jerryji1993/DNABERT#102]
Here is the location file:/src/transformer/modeling_utils.py

  192      exteneded_attention_mask = extended_attention_mask.to(dtype=self.dtype)
  193      exteneded_attention_mask = (1.0 - extended_attention_mask) * -10000.0
  194     return extended_attention_mask

The returned variable name is wrong！！！
